core/test/surplus-test.ts

```diff
@@ -96,4 +111,15 @@ describe('Surplus Auction', () => {
     it('forbids fetching inexistant auctions', async () => {
         expect(fetchSurplusAuctionByIndex('custom', 3333)).to.be.revertedWith('No active auction exists with this id');
     });
+    it('calculates the minimum bid increase', async () => {
+        const auctions = await fetchActiveSurplusAuctions('custom');
+        expect(auctions[0].id).to.equal(2328);
+
+        const bid = await getNextMinimumBid('custom', auctions[0]);
+        if (!bid || !auctions[0].bidAmountMKR) {
+            throw new Error('Expected values were not defined.');
+        }
+        expect(bid.toString()).to.equal('16.94241213279722952');
```

Comment:
There are actually can be a problem that this value is rounded on a wrong way. And in order to properly test it we need to try to bid on the auction with this value (and don't receive an error)